ted_sws/notice_transformer/entrypoints/cmd_mapping_suite_transformer.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/python3
+
+import os
+import sys
+
+module_path = os.path.dirname(os.path.realpath(__file__)) + '/../../../'
```

Comment:
This looks very dangerous and easy to break. 
One shall rather use `pkg_resources` following the example from metadata_normaliser.

```
try:
    import importlib.resources as pkg_resources
except ImportError:
    # Try backported to PY<37 `importlib_resources`.
    import importlib_resources as pkg_resources

```

Reply:
This was added to be run without python package manager (to get used faster by semantic team). It will be updated, so it runs as a package command